common/button/button-en.html

```diff
@@ -0,0 +1,415 @@
+	<li><a href="#activestate">Active State</a></li>
+	<li><a href="#disabledstate">Disabled State</a></li>
+	<li><a href="#bghorizontal">Button Groups (Horizontal)</a></li>
+	<li><a href="#bgvertical">Button Groups (Vertical)</a></li>
+	<li><a href="#buttontoolbar">Button Toolbar</a></li>
+	<li><a href="#inputgroups">Input Groups</a></li>
+	<li><a href="#datatoggle">Data-toggle Buttons</a></li>
+	</ul></li>
+</ul>
+
+<h2 id="purpose">Purpose</h2>
+<p>Use buttons to create a call to action that entices the user to submit content from a form. Also, use buttons to take users to a webpage that provides additional information.</p>
+
+
+
+<h2 id="basic">Basic use</h2>
+
+<h3>Default</h3>
+<p>Use the button classes on <code>a</code>, <code>button</code>, or <code>input</code> elements to create visual consistency. The browser shows certain elements as a button. However, there is often a lack of visual consistency for each button type. Each browser can have a different button default appearance.</p>
+
+<h4>Working example</h4>
+<h5>Default:</h5>
+<button type="submit">Button</button>
+<input type="button" value="Input" />
+<input type="submit" value="Submit" />
+<a href="#" >Link</a>
+
+<h5>Styled:</h5>
+<button class="btn btn-default" type="submit">Button</button>
+<input class="btn btn-default" type="button" value="Input" />
+<input class="btn btn-default" type="submit" value="Submit" />
+<a class="btn btn-default" href="#" role="button">Link</a>
+
+<h4>Code Sample</h4>
+<pre><code>
+	<span class="com">// Default button:</span>
+	&lt;button type="submit"&gt;
+	Button
+	&lt;/button&gt;
+	&lt;input type="button" value="Input"/&gt;
+	&lt;input type="submit" value="Submit"/&gt;
+	&lt;a href="#" role="button"&gt;
+	Link
+	&lt;/a&gt;
+
+	<span class="com">// Styled button:</span>
+	&lt;button class="<strong>btn btn-default</strong>" type="submit"&gt;
+	Button
+	&lt;/button&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="button" value="Input"/&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="submit" value="Submit"/&gt;
+	&lt;a class="<strong>btn btn-default</strong>" href="#" role="button"&gt;
+	Link
+	&lt;/a&gt;
+</code></pre>
+
+<h2 id="enhanced">Enhanced use</h2>
+
+<h3 id="colours/styles">Colours/Styles</h3>
+<p>Use to add colours to buttons. Colours signify various things such as a link to warning information (caution button) to a link to the next learning module (success button).</p>
+<h4>Working example</h4>
+<button type="button" class="btn btn-default">Default</button>
+<button type="button" class="btn btn-primary">Primary</button>
+<button type="button" class="btn btn-success">Success</button>
+<button type="button" class="btn btn-info">Info</button>
+<button type="button" class="btn btn-warning">Warning</button>
+<button type="button" class="btn btn-danger">Danger</button>
+<button type="button" class="btn btn-link">Link</button>
+
+<h4>Code sample</h4>
+<pre><code>
+	<span class="com">// Default:</span>
```

Comment:
Remove that span everywhere.

```suggestion
	// Default:
```
or
```suggestion
	&lt;!-- Default: -->
```


Instead you can enhance this page with wb-pretty plugin to colorify the code sample

Reply:
Addressed in commit  "Button: Addressed Review Comments"